### ISSUE TITLE
Implement Horizontal Layouts

### ISSUE THREAD

**Issue by CryZe** [LiveSplit/LiveSplitOne#159]
livesplit-core and LiveSplit One on the Desktop / Mobile just gained support for horizontal layouts. LiveSplit One on the Web still needs to implement additional support for it, because in the web we transform everything into HTML + CSS, which isn't done yet.

**Comment by ckurtz22** [comment]
You say that LiveSplit One on desktop supports horizontal layouts, but where is LiveSplit One for the desktop? The only version of LiveSplit One that I see is this repository which seems to be web based only?

**Comment by CryZe** [comment]
It's not entirely public yet as it missing pretty much all the UI. There is no Run Editor, Layout Editor, Settings or anything. The only thing it is showing at the moment is the actual timer. All the settings are done via a config file. An [early experimental repository](https://github.com/CryZe/livesplit-one-desktop) already exists that you could build if you wanted to, but it's not the final repository. I'm probably setting up the real repository soon, at which point I'll also rename this one to LiveSplitOneWeb or so.

**Comment by Tallestthomas** [comment]
Is this still an open issue? I can take a crack at this and probably knock it out fairly quick.

**Comment by CryZe** [comment]
Yeah this is still not implemented. Feel free to take this. If you run into any problems, feel free to ask for help (possibly via Discord DMs so it's faster)

**Comment by W-R-Ramirez** [comment]
@CryZe HI I was wondering what the status on this was?

**Comment by CryZe** [comment]
The status is actually explained in detail here: https://github.com/LiveSplit/LiveSplitOne/issues/286#issuecomment-984516384

TL;DR: We are switching to canvas using the same code that the Desktop version is using, which automatically comes with horizontal layout support.

**Comment by W-R-Ramirez** [comment]
Awesome thanks! Is there anywhere I can follow the progress besides just checking the repo every so often?

**Comment by CryZe** [comment]
There's a livesplit-one channel in the [Discord](https://discord.gg/0nuoLLvyUPPjq1YW) where I post status updates. Other than that you can get notifications from Github.
